react/components/BuyTogether/index.tsx

```diff
@@ -115,7 +115,7 @@ const BuyTogether: StorefrontFunctionComponent<Props> = ({
       if (currentItem.selectedItem.seller?.commertialOffer.Price) {
         return total + currentItem.selectedItem.seller.commertialOffer?.Price
       }
-      return total + currentItem.selectedItem.sellers[0].commertialOffer?.Price
+      return total + currentItem.selectedItem.sellers[0]?.commertialOffer?.Price
```

Comment:
Is it possible that this value is `NaN`?

Reply:
yes. but it will be fixed soon on the backend. I added this check just to make sure the app wouldn't break until the backend is ok.